apps/allocations/contracts/Allocations.sol

```diff
@@ -162,16 +162,21 @@ contract Allocations is AragonApp, Fundable {
         payout.balance = 0;
         FundForwarder fund = new FundForwarder(payoutId, address(this));
         payout.proxy = address(fund);
-        emit NewAccount(payoutId); 
+        emit NewAccount(payoutId);
     }
 
-    function fund(uint256 id) external payable { 
+    function fund(uint256 id) external payable {
         Payout storage payout = payouts[id];
         require(!payout.informational);
         payout.balance = payout.balance.add(msg.value);
         emit FundAccount(id);
     }
 
+    /**
+    * @dev This function distributes the payouts to the candidates in accordance with the distribution values
+    * @notice Send payout amounts to the candidates in accordance with the distribution proportions
+    * @param _payoutId Any relevent label for the payout
+    */
     function runPayout(uint256 _payoutId) external payable isInitialized auth(EXECUTE_PAYOUT_ROLE) returns(bool success) {
```

Comment:
Where in the UI is this?

Reply:
This shows up when you begin the distribute payout/runpayout transaction